src/visitpy/visitpy/PyView2DAttributes.C

```diff
@@ -620,174 +621,6 @@ View2DAttributes_str(PyObject *v)
-
-// Good ref on python 2 and 3 changes for PyNumberMethods
-// https://py3c.readthedocs.io/en/latest/ext-types.html#pynumbermethods
-
-// PYTHON 2 STRUCT FOR PyNumberMethods
-
-typedef struct {
-     binaryfunc nb_add;
-     binaryfunc nb_subtract;
-     binaryfunc nb_multiply;
-     binaryfunc nb_divide;  // MISSING IN PYTHON 3
-     binaryfunc nb_remainder;
-     binaryfunc nb_divmod;
-     ternaryfunc nb_power;
-     unaryfunc nb_negative;
-     unaryfunc nb_positive;
-     unaryfunc nb_absolute;
-     inquiry nb_nonzero;       // Used by PyObject_IsTrue 
-     unaryfunc nb_invert;
-     binaryfunc nb_lshift;
-     binaryfunc nb_rshift;
-     binaryfunc nb_and;
-     binaryfunc nb_xor;
-     binaryfunc nb_or;
-     coercion nb_coerce;     // MISSING IN PYTHON 3   // Used by the coerce() function 
-     unaryfunc nb_int;
-     unaryfunc nb_long;   // MUST BE NULL IN PYTHON 3
-     unaryfunc nb_float;
-     unaryfunc nb_oct;    // MISSING IN PYTHON 3
-     unaryfunc nb_hex;    // MISSING IN PYTHON 3
-
-     // Added in release 2.0 
-     binaryfunc nb_inplace_add;
-     binaryfunc nb_inplace_subtract;
-     binaryfunc nb_inplace_multiply;
-     binaryfunc nb_inplace_divide;  // MISSING IN PYTHON 3
-     binaryfunc nb_inplace_remainder;
-     ternaryfunc nb_inplace_power;
-     binaryfunc nb_inplace_lshift;
-     binaryfunc nb_inplace_rshift;
-     binaryfunc nb_inplace_and;
-     binaryfunc nb_inplace_xor;
-     binaryfunc nb_inplace_or;
-
-     // Added in release 2.2 
-     binaryfunc nb_floor_divide;
-     binaryfunc nb_true_divide;
-     binaryfunc nb_inplace_floor_divide;
-     binaryfunc nb_inplace_true_divide;
-
-     // Added in release 2.5 
-     unaryfunc nb_index;
-} PyNumberMethods;
-
-// PYTHON 3 STRUCT FOR PyNumberMethods
-
-typedef struct {
-     binaryfunc nb_add;
-     binaryfunc nb_subtract;
-     binaryfunc nb_multiply;
-     binaryfunc nb_remainder;
-     binaryfunc nb_divmod;
-     ternaryfunc nb_power;
-     unaryfunc nb_negative;
-     unaryfunc nb_positive;
-     unaryfunc nb_absolute;
-     inquiry nb_bool;
-     unaryfunc nb_invert;
-     binaryfunc nb_lshift;
-     binaryfunc nb_rshift;
-     binaryfunc nb_and;
-     binaryfunc nb_xor;
-     binaryfunc nb_or;
-     unaryfunc nb_int;
-     void *nb_reserved;
-     unaryfunc nb_float;
-
-     binaryfunc nb_inplace_add;
-     binaryfunc nb_inplace_subtract;
-     binaryfunc nb_inplace_multiply;
-     binaryfunc nb_inplace_remainder;
-     ternaryfunc nb_inplace_power;
-     binaryfunc nb_inplace_lshift;
-     binaryfunc nb_inplace_rshift;
-     binaryfunc nb_inplace_and;
-     binaryfunc nb_inplace_xor;
-     binaryfunc nb_inplace_or;
-
-     binaryfunc nb_floor_divide;
-     binaryfunc nb_true_divide;
-     binaryfunc nb_inplace_floor_divide;
-     binaryfunc nb_inplace_true_divide;
-
-     unaryfunc nb_index;
-
-     binaryfunc nb_matrix_multiply; // MISSING IN PYTHON 2
-     binaryfunc nb_inplace_matrix_multiply; // MISSING IN PYTHON 2
-} PyNumberMethods;
-
-*/
-
-//
```

Comment:
Why was all this removed? (Similar question for PyView3DAttributes)

Reply:
Hmmmm....Will have to look. I don't think I intentionally removed anything here.